script.skipit/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.skipit" name="SkipIt" version="1.0.0" provider-name="shartrec">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+	</requires>
+	<extension point="xbmc.python.script" library="addon.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<platform>all</platform>
+		<summary lang="en">Skip It</summary>
+		<description lang="en">Quick skip over Ads. This addon is based on the ideas from the original Topfield SkipIt TAP. It uses a binary search technique to quickly get you to the place you want in a video.</description>
+		<license>GNU General Public License, v2</license>
+		<language></language>
+<!-- 		<forum>http://forum.kodi.tv/showthread.php?tid=209948</forum> -->
```

Comment:
Please fill the below values appropriately

Reply:
Thanks for the feedback.
I'm not sure how I should fill in some of these.
Forum : Should I create a thread a forum thread now and point there, or just point to the overall add-on forum?
Source : I have a fork of repo-scripts, but should the source point there or here (i.e.  xbmc /repo-scripts ) or do I need a seperate standalone repo with just my add-on (that seems a bit wrong to me)?

Reply:
You can remove the forum and website one. For source you should have a GitHub repo for your add-on so that users know where the upstream is

Reply:
Set to upstream repo